plugins/Actions/API.php

```diff
@@ -506,18 +533,21 @@ function ($visits) {
      */
     private function filterNonExitActions($dataTable)
     {
-        $dataTable->filter('ColumnCallbackDeleteRow',
-            array(PiwikMetrics::INDEX_PAGE_EXIT_NB_VISITS,
-                  function ($visits) {
-                      return !strlen($visits);
-                  })
+        $dataTable->filter(
+            'ColumnCallbackDeleteRow',
+            [
+                PiwikMetrics::INDEX_PAGE_EXIT_NB_VISITS,
+                function ($visits) {
+                    return !strlen($visits);
+                }
+            ]
         );
     }
 
     private function addPageProcessedMetrics(DataTable\DataTableInterface $dataTable)
     {
         $dataTable->filter(function (DataTable $table) {
-            $extraProcessedMetrics = $table->getMetadata(DataTable::EXTRA_PROCESSED_METRICS_METADATA_NAME);
+            $extraProcessedMetrics = $table->getMetadata(DataTable::EXTRA_PROCESSED_METRICS_METADATA_NAME) ?: [];
```

Comment:
Be good if this PR was two commits so I could easily find the actual changes amongst the formatting improvements.